lib/kvdb/kvdb_rparams.c

```diff
@@ -651,6 +651,27 @@ static const struct param_spec pspecs[] = {
             },
         },
     },
+    {
+        .ps_name = "cndb_compact_hwm",
+        .ps_description = "High water mark percentage (x100)",
+        .ps_flags = PARAM_FLAG_EXPERIMENTAL,
+        .ps_type = PARAM_TYPE_U32,
+        .ps_offset = offsetof(struct kvdb_rparams, cndb_compact_hwm),
+        .ps_size = PARAM_SZ(struct kvdb_rparams, cndb_compact_hwm),
+        .ps_convert = param_default_converter,
+        .ps_validate = param_default_validator,
+        .ps_stringify = param_default_stringify,
+        .ps_jsonify = param_default_jsonify,
+        .ps_default_value = {
+            .as_uscalar = HSE_CNDB_COMPACT_HWM_DEFAULT,
+        },
+        .ps_bounds = {
+            .as_uscalar = {
+                .ps_min = 0,
+                .ps_max = 100 * 100,
+            },
+        },
+    },
```

Comment:
Should I push up my PR to support config params of type `double`?  Then we wouldn't need these scaled integer parameters.

Reply:
Sure. I figured people just didn't like doubles in the code base.

Reply:
Yeah, a double would make this cleaner.

Reply:
We couldn't use them when had plans to build HSE as a kernel module.   I just pushed up a PR.

Reply:
It would be good to identify what other things could be moved to doubles